Include/pyport.h

```diff
@@ -406,7 +406,7 @@ extern "C" {
 #endif
 
 /* get and set x87 control word for VisualStudio/x86 */
-#if defined(_MSC_VER) && !defined(_WIN64) /* x87 not supported in 64-bit */
+#if defined(_MSC_VER) && !defined(_WIN64) && !defined(_M_ARM) /* x87 not supported in 64-bit */
```

Comment:
Seems good. But don't see activity in the issue on bugs.python.org of this change.

Reply:
Should probably test `_M_IX86` instead (and drop the `_WIN64` check). See [predefined macros on MSVC](https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=vs-2017).

Also cc @zooba.